packages/keybr-keyboard/lib/layout.ts

```diff
@@ -890,9 +890,54 @@ export class Layout implements XEnumItem {
       Geometry.MATRIX,
     ),
   );
+  static readonly TH_TH = new Layout(
+    /* id= */ "th-th",
+    /* xid= */ 0x9a,
+    /* name= */ "Thai Kedmanee",
+    /* family= */ "qwerty",
+    /* language= */ Language.TH,
+    /* emulate= */ true,
+    /* geometries= */ new Enum(
+      Geometry.ANSI_101,
+      Geometry.ANSI_101_FULL,
+      Geometry.ISO_102,
+      Geometry.ISO_102_FULL,
+      Geometry.MATRIX,
+    ),
+  );
+  static readonly TH_PAT = new Layout(
+    /* id= */ "th-pat",
```

Comment:
Maybe this id should be `"th-th-pat"`? (Just like the layout definition file is named)